sdk/nodejs/iam/groupPolicyAttachment.ts

```diff
@@ -61,7 +61,7 @@ export class GroupPolicyAttachment extends pulumi.CustomResource {
     /**
      * The group the policy should be applied to
      */
-    public readonly group!: pulumi.Output<Group>;
+    public readonly group!: pulumi.Output<string>;
```

Comment:
Note that although this is a typescript level breaking change - any code which no longer type checks would not have worked correctly as the true value was always a string.

Reply:
I am so glad you mentioned this because I had no idea why it changed to string as the output - I just didn't want to bother you tonight with it :) Thanks for that! It really helps me